src/main/java/com/endava/internship/warmup/domain/service/ArrayProcessorWithForLoops.java

```diff
@@ -34,7 +49,15 @@ public boolean someMatch(final int[] input, IntPredicate predicate) {
     public boolean allMatch(final String[] input,
                             ToIntFunction<String> function,
                             IntPredicate predicate) {
-        throw new UnsupportedOperationException("Method not implemented yet");
+        for(String s : input) {
+            int num = function.applyAsInt(s);
+
```

Comment:
Here also there is not need of a empty line, here is not so much code to be not enough readable